README.md

```diff
@@ -298,6 +298,42 @@ const App = () => {
 export default App;
 ```
 
+### [io-ts](https://github.com/gcanti/io-ts)
+
+Validate your data with powerful decoders.
+
+[![npm](https://img.shields.io/bundlephobia/minzip/io-ts?style=for-the-badge)](https://bundlephobia.com/result?p=io-ts@2.2.16)
+
+```typescript jsx
+import React from 'react';
+import { useForm } from 'react-hook-form';
+import { ioTsResolver } from '@hookform/resolvers/io-ts';
+import t from 'io-ts';
+// you don't have to use io-ts-types but it's very useful
+import tt from 'io-ts-types';
```

Comment:
minor thoughts: perhaps we want to keep the example simple without introducing another lib?

Reply:
Yeah, I wondered about wether I should use it or not... `io-ts` by itself is very low-level. So I though "hum maybe it'll be scary for newcomers". But I can make it simpler for sure. It's just if you wan't a nice validator like `min` or `max` you have to craft it yourself !

Reply:
I don’t mind either 👌just a minor thought.